Remotion/Validation/Core/Attributes/Validation/LengthValidationAttribute.cs

```diff
@@ -51,6 +52,9 @@ protected override IEnumerable<IPropertyValidator> GetValidators (IPropertyInfor
       ArgumentUtility.CheckNotNull("property", property);
       ArgumentUtility.CheckNotNull("validationMessageFactory", validationMessageFactory);
 
+      if (MinLength == 0 && MaxLength == 0)
+        return Enumerable.Empty<IPropertyValidator>();
```

Comment:
unit test please.
also, we should integrate this into the IF/ELSE tree. it's part of the remaining rule set and if we had thought of it before, we wouldn't have killed the original ELSE.

Reply:
I hope you mean the IF/ELSE tree in `GetValidators()` as I made the changes now, because if you meant the IF/ELSE tree in `CreateValidator()`, I am not sure if it is necessary to do a null-check and then return empty array in the outer method

Reply:
Yeah, I actually meant `CreateValidator()` but I didn't realize that this would lead to furthor problems. I'll do the actual review tomorrow. Thanks for the feedback on what works and what doesn't!